cnn_framework/predictor.py

```diff
@@ -414,7 +414,7 @@ def save_model(self, loss, inner_val_loss, mean_outer_val_loss, mean_test_loss,
             np.savez(fpath + '_mean_std.npz', mean=self.y_mean, std=self.y_std)
             logging.info('...saved y mean and standard deviation to {}_mean_std.npz'.format(fpath))
 
-    def predict(self, molecule, molecule_tensor=None, sigma=False):
+    def predict(self, molecule=None, molecule_tensor=None, sigma=False):
```

Comment:
Perhaps we should raise an exception if neither `molecule` nor `molecule_tensor` are specified.

Reply:
Done